srcpkgs/lc0/template

```diff
@@ -0,0 +1,35 @@
+# Template file for 'lc0'
+pkgname=lc0
+version=0.27.0
+revision=1
+build_style=meson
+configure_args="-Dgtest=false"
+_lczero_commit=00fd892e648160c294346c87449126d9bad80a16
+hostmakedepends="tar pkg-config"
```

Comment:
Last nit, does it actually need `tar` here? It shouldn't be using any subproject, after all.

Reply:
Done